lib/node/nodes/kmeans.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var Node = require('../node');
+var dot = require('dot');
+
+var TYPE = 'kmeans';
+
+var PARAMS = {
+    source : Node.PARAM.NODE(Node.GEOMETRY.POINT),
+    clusters : Node.PARAM.NUMBER
+};
+
+var KMeans  = Node.create(TYPE, PARAMS, {cache: false});
```

Comment:
for the moment we just need to set `cache: true` so we avoid postgres planner to make wrong decisions.

In the future we will see